.travis.yml

```diff
@@ -189,6 +206,45 @@ matrix:
         - docker run -d --network host --name symbolicator us.gcr.io/sentryio/symbolicator:latest run
         - docker ps -a
 
+    - <<: *postgres_default
+      name: 'Sentry-Relay integration tests'
+      env: TEST_SUITE=relay-integration DB=postgres
+      services:
+        - docker
+        - memcached
+        - redis-server
+        - postgresql
+      before_install:
+        - *before_install_default
+        - docker pull us.gcr.io/sentryio/relay:latest # pull relay we'll run and kill it for each test
+
+        - > # clickhouse
+          docker run
+          --name sentry_clickhouse
+          -d --network host
+          -v  sentry_clickhouse:/var/lib/clickhouse
```

Comment:
_nit_: This and the line `-e CLICKHOUSE_CONFIG=/etc/clickhouse-server/config.xml` may not be needed in travis. That makes the configuration equivalent with the one in Snuba 👍